cypress/page/payment-step.page.ts

```diff
@@ -1,10 +1,11 @@
 class PaymentStepPage {
     private payButton: string;
     private confirmOrderMessage: string;
+    
 
     constructor() {
-        this.payButton = "a.bankwire";
-        this.confirmOrderMessage = "#cart_navigation>button>span";
+        this.payButton = "a[title='Pay by bank wire'].bankwire";
+        this.confirmOrderMessage = "#cart_navigation .button span";
```

Comment:
try not to use span, in this case you identify the button because is of type submit (the form should have this)